frontend/src/pages/ReviewPage.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { ReactElement, useEffect, useState } from 'react';
+import { Box, Container, Typography, makeStyles } from '@material-ui/core';
+import { get } from '../utils/call';
+import ApartmentCards from '../components/ApartmentCard/ApartmentCards';
+import { CardData } from '../App';
+
+const ReviewPage = (): ReactElement => {
+  const useStyles = makeStyles({
+    rentingBox: {
+      marginTop: '3em',
+      marginBottom: '2em',
+    },
+    rentingText: {
+      fontWeight: 500,
+    },
+  });
+
+  const classes = useStyles();
+  const [buildingData, setBuildingData] = useState<CardData[]>([]);
```

Comment:
Might want to rename `buildingData` here to something else because `buildingData` is the name of a property in `CardData`.